packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -694,6 +694,10 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		const maybeUnusedStorageIds: Set<string> = new Set();
 		for (const route of blobRoutes) {
 			const blobId = getBlobIdFromGCNodePath(route);
+			assert(
```

Comment:
@ChumpChief @tylerbutler @jason-ha does asserting here or in this package in general affect perf?